spec/models/user_model_spec.rb

```diff
@@ -47,6 +47,7 @@
 
   describe "ClassMethods" do
     before :all do
+      DatabaseCleaner.start
```

Comment:
It probably would work with bigger changes to the tests, but that's way out of scope.